palette/src/luma/relative_contrast.rs

```diff
@@ -0,0 +1,34 @@
+pub mod wcag {
```

Comment:
Not sure if the double wrapping is necessary, unless you have plans for suggesting more ways of implementing contrast ratio. :slightly_smiling_face:

Reply:
I did this preemptively to explore other contrast types later. After you pointed it out and I looked it up, I realized there weren't really other ways.

Reply:
I think we can solve that problem later if it appears.